mantis-server/mantis-server-worker-client/build.gradle

```diff
@@ -18,6 +18,8 @@ dependencies {
     api project(":mantis-control-plane:mantis-control-plane-core")
     api project(":mantis-control-plane:mantis-control-plane-client")
 
+    implementation libraries.spectatorApi
```

Comment:
@hmit should we still have spectator api?

Reply:
Adding some context: the build was failing without this.

Reply:
yes it's still needed